extensions/wikia/WikiaApi/WikiaApiQueryAllUsers.php

```diff
@@ -376,9 +376,57 @@ private function local_users() {
 
 		$result->setIndexedTagName_internal(array('query', $this->getModuleName()), 'u');
 	}
+
+	private function getBaseAllowedParams() {
+		$userGroups = User::getAllGroups();
+		return array(
+			'from' => null,
+			'to' => null,
+			'prefix' => null,
+			'dir' => array(
+				ApiBase::PARAM_DFLT => 'ascending',
+				ApiBase::PARAM_TYPE => array(
+					'ascending',
+					'descending'
+				),
+			),
+			'group' => array(
+				ApiBase::PARAM_TYPE => $userGroups,
+				ApiBase::PARAM_ISMULTI => true,
+			),
+			'excludegroup' => array(
+				ApiBase::PARAM_TYPE => $userGroups,
+				ApiBase::PARAM_ISMULTI => true,
+			),
+			'rights' => array(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "rights" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)